docs/versioned_docs/version-v1.3.0/03-Guides/migration-guide-v1.0.md

```diff
@@ -0,0 +1,96 @@
+# Migration guide towards v1.0
+Starting from v1.0, there're some major breaking changes. To make it easier for you you migrate towards this new version, we have assembled an migration guide to help you in the process.
+
+## New Azure SDK
+We have chosen to also update our library to the new Azure SDK when interacting with the Azure Service Bus ([#159](https://github.com/arcus-azure/arcus.messaging/discussions/159)). This package update has some consequences on our library.
+
+> Note that Azure Service Bus plug-ins are [still not supported](https://github.com/arcus-azure/arcus.messaging/discussions/159) in this new SDK.
```

Comment:
I'm fine with having this warning here as opposed to have it on the 'index' page.
People that read this part, know that they might have to pay attention to this when migrating, so ok.